app/src/main/java/vn/mbm/phimp/me/base/BaseActivity.java

```diff
@@ -43,14 +43,19 @@ public void onPause() {
     public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
         switch (item.getItemId()) {
             case R.id.navigation_home:
-                startActivity(new Intent(this, LFMainActivity.class));
-                finish();
+                if (getNavigationMenuItemId() != R.id.navigation_home) {
```

Comment:
Why checking each item by name, if you can get this by `item.getItemId()`. It will make code not redundant. 
add this check in the beginning and execute `switch` block on the basis of that.